src/main/scala/ai/privado/passes/PropertyParserPass.scala

```diff
@@ -169,30 +175,37 @@ class PropertyParserPass(cpg: Cpg, projectRoot: String, ruleCache: RuleCache, la
   }
 
   private def loadAndConvertYMLtoProperties(file: String): List[(String, String)] = {
-    parser.parse(better.files.File(file).contentAsString) match {
-      case Right(json) => {
-        try {
-          JsonFlattener
-            .flattenAsMap(json.toString)
-            .asScala
-            .toList
-            .map(p => {
-              if (p._2 == null)
-                ("", "")
-              else
-                (p._1, p._2.asInstanceOf[String])
-            })
-        } catch {
-          case e: Throwable =>
-            logger.trace(s"Error while creating properties node for file $file")
-            logger.debug(s"Error while creating properties node for file : $file, error : ${e.getMessage}")
-            List[("", "")]()
-        }
-      }
-      case Left(error) => {
-        List[("", "")]()
+
```

Comment:
as linenumber is also returned along with the value part here, can't we change the output to (String, String, linenumber) instead ?

Reply:
Ah yes, makes sense.